django_cloud_deploy/skeleton/templates/settings_template/extra_settings.py-tpl

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .base_settings import *
+
+
+if 'INSTALLED_APPS' in locals():
```

Comment:
But, we don't install our app if INSTALLED_APPS is not defined?

Reply:
I modified the file to make it handle undefined 'INSTALLED_APPS'.